src/core/queue/__tests__/event-queue.test.ts

```diff
@@ -422,6 +422,35 @@ describe('Flushing', () => {
       expect(amplitude.track).toHaveBeenCalled()
     })
 
+    test('delivers to destinations that exist as an object', async () => {
+      const eq = new EventQueue()
+
+      jest.spyOn(amplitude, 'track')
+
+      const evt = {
+        type: 'track' as const,
+        integrations: {
+          All: false,
+          Amplitude: {
+            amplitudeKey: 'foo',
+          },
+        },
+      }
```

Comment:
do we need to update any type definitions?

Reply:
i don't believe so

Reply:
Just checked that `Integrations` already accepts `JSONObjects` https://github.com/segmentio/analytics-next/blob/c13a3053afb8f2f2ce96e6d617a357a1cea83c1d/src/core/events/interfaces.ts#L11-L12